README.md

```diff
@@ -248,6 +248,7 @@ Available Commands:
   argocd                  Install argocd
   cert-manager            Install cert-manager
   chart                   Install the specified helm chart
+  consul                  Install Consul
```

Comment:
Is this consul as in consul but on Kubernetes, or "consul-connect" as in the service mesh?

Reply:
@jsiebens PTAL

Reply:
By default Consul Connect is enabled by the Helm chart. 
Will change it to reflect that...